auto_round/compressors/base.py

```diff
@@ -2471,13 +2475,30 @@ def _get_current_num_elm(
         current_input_ids = [input_ids[i] for i in indices]
         return sum(id.numel() for id in current_input_ids)
 
+    def quantize_block(
+        self,
+        block: torch.nn.Module,
+        inputs: tuple[Union[list[torch.Tensor], dict, Any], Optional[dict]],
+        q_input: Union[torch.Tensor, dict, None] = None,
+        device: Union[str, torch.device] = "cpu",
+        auto_offload=True,
+    ):
+        # TODO: release below assertion after supporting MLLM and diffusion model quantization with quantize_block
```

Comment:
could you add a comment that this function is mainly for llmc

Reply:
Thanks, added.